Smith_Ben/Algorithms/pg50(removeblanks, acronyms).html

```diff
@@ -0,0 +1,41 @@
+<html>
+<head>
+  <title></title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <script type="text/javascript">
+  /*
+  $(document).ready(function(){
+    // ... your assignment here
+  });
+  */
+
+  function(remove blanks){
+    var new = ' '
+    For(var i = 0; i < str.length; i++){
+      if(str[i] == ' '){
+      }
+      else{
+        new += str[i];
+      }
+    }
+    return new;
+  }
+
+  function(acronyms){
+    var split_arr = arr.split(' ');
+    var first_letter = [];
+    for(var i=0; i<split_arr.length(); i++){
+      first_letter.push(toUpperCase(split_arr[i][0]))
+    }
+    return first_letter.join('')
+  }
+  
```

Comment:
after writing these algos, be sure to run them (e.g. call them) to test that they actually worx